presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -38,23 +40,31 @@ protected void setup(Binder binder)
 
         if (buildConfigObject(HiveConfig.class).getRecordingPath() != null) {
             binder.bind(HiveMetastore.class)
-                    .annotatedWith(ForRecordingHiveMetastore.class)
-                    .to(GlueHiveMetastore.class)
-                    .in(Scopes.SINGLETON);
+                .annotatedWith(ForRecordingHiveMetastore.class)
+                .to(GlueHiveMetastore.class)
+                .in(Scopes.SINGLETON);
```

Comment:
Let's revert this change since the original indent is right. Also, we should separate such commits to correspond with commit message when needed. You can install codestyle from here. https://github.com/airlift/codestyle

Reply:
@ebyhr Added original indentation.